test/skate/detours/missed_stops_test.exs

```diff
@@ -0,0 +1,99 @@
+defmodule Skate.Detours.MissedStopsTest do
+  use ExUnit.Case
+
+  alias Util.Location
+
+  describe "missed_stops" do
+    test "given a straight line, should return missed stops" do
+      param = %Skate.Detours.MissedStops{
+        connection_start: Location.new(-0.001, 1),
```

Comment:
**suggestion (non-blocking):** I wonder if little ASCII art diagrams for these different test cases would make them easier to read - obviously they wouldn't be to scale, but I wonder if you could at least capture the relative positions of points and stops to give a sense of what's going on.

Reply:
Thoughts?
```ex
      ##                                                 (Missed Stop)
      ##        Stops: v(0.001, 0)                        v(0.001, 5)   v(0.001, 5)
      ##               o                                  o             o
      ##
      ##               o------o------o------o------o------o------o------o
      ## Shape Points: ^(0,0) ^(0,1) ^(0,2) ^(0,3) ^(0,4) ^(0,5) ^(0,6) ^(0,7)
      ##                                                        
      ## Connection Points :  o                                  o
      ##                      ^(-0.001, 1)                       ^(-0.00, 6)
      ##                   (connection_start)                  (connection_end)
```

Reply:
I don't care what anyone else says - this is art.

(Nit: I think `connection_end` in the diagram should be `-0.001` rather than `-0.00`)

Reply:
I can continue making diagrams like this, but it was pretty tricky and annoying to make this (let alone maintain it??) and took me longer than I'd have liked.

And it's only gonna get more difficult with the "cross" test and others.

Thoughts on maybe using Excalidraw? (or miro)
Example: https://excalidraw.com/#json=OrMM928mw4CR3Qy8sc6oL,sXiFCU1s-K1ugEQvgSvh3g

I'm not happy with the idea of having this diagram hosted by an external service, so while a link would probably be the easiest option if we used Excalidraw, we could instead save the diagrams as SVG, PNG, or Export the `.excalidraw` (json) document and version it into Skate.

Reply:
Updated visual https://excalidraw.com/#json=91bSifAkQwmz3uy4x8k02,Eu4K8LXTkYOtPcBchBjkUg

Reply:
Yeah, if you think it's too tricky don't worry about it. I figured it would be a neat idea to try but I don't think it's worth sinking too much time into.

Reply:
I do like the idea of adding a diagram though, do y'all have issues with using Excalidraw/preferences on how(link, svg, png, `.excalidraw`) it's used?

Reply:
I'm personally okay with using links for now, and I think I'd otherwise prefer SVG to make it easy to view, or `.excalidraw` for easy to edit.

Reply:
My concern with having an external link or something like an SVG in the repo is that it would also be hard to maintain.